node/chainSimulator/dtos/state.go

```diff
@@ -0,0 +1,17 @@
+package dtos
+
+// AddressState will hold the address state
+type AddressState struct {
+	Address string `json:"address"`
+	// ShardID: This field is needed for the system account address (it is the same on all shards).
+	ShardID          uint32            `json:"shardID,omitempty"`
```

Comment:
I think we do not need this. The data on all SystemAccounts (on all shards) should be the same.

Reply:
removed